src/contracts/core/StrategyManager.sol

```diff
@@ -262,90 +265,93 @@ contract StrategyManager is
-     * @param staker The address to add shares to
+     * @notice This function adds `scaledShares` for a given `strategy` to the `staker` and runs through the necessary update logic.
+     * @param staker The address to add scaledShares to
      * @param token The token that is being deposited (used for indexing)
-     * @param strategy The Strategy in which the `staker` is receiving shares
-     * @param shares The amount of shares to grant to the `staker`
-     * @dev In particular, this function calls `delegation.increaseDelegatedShares(staker, strategy, shares)` to ensure that all
-     * delegated shares are tracked, increases the stored share amount in `stakerStrategyShares[staker][strategy]`, and adds `strategy`
+     * @param strategy The Strategy in which the `staker` is receiving scaledShares
+     * @param scaledShares The amount of scaled shares to grant to the `staker`
+     * @dev In particular, this function calls `delegation.increaseDelegatedScaledShares(staker, strategy, scaledShares)` to ensure that all
+     * delegated scaledShares are tracked, increases the stored share amount in `stakerStrategyScaledShares[staker][strategy]`, and adds `strategy`
      * to the `staker`'s list of strategies, if it is not in the list already.
      */
-    function _addShares(address staker, IERC20 token, IStrategy strategy, uint256 shares) internal {
+    function _addScaledShares(address staker, IERC20 token, IStrategy strategy, uint256 scaledShares) internal {
         // sanity checks on inputs
-        require(staker != address(0), "StrategyManager._addShares: staker cannot be zero address");
-        require(shares != 0, "StrategyManager._addShares: shares should not be zero!");
+        require(staker != address(0), "StrategyManager._addScaledShares: staker cannot be zero address");
+        require(scaledShares != 0, "StrategyManager._addScaledShares: shares should not be zero!");
 
-        // if they dont have existing shares of this strategy, add it to their strats
-        if (stakerStrategyShares[staker][strategy] == 0) {
+        // if they dont have existing scaled shares of this strategy, add it to their strats
+        if (stakerStrategyScaledShares[staker][strategy] == 0) {
             require(
                 stakerStrategyList[staker].length < MAX_STAKER_STRATEGY_LIST_LENGTH,
-                "StrategyManager._addShares: deposit would exceed MAX_STAKER_STRATEGY_LIST_LENGTH"
+                "StrategyManager._addScaledShares: deposit would exceed MAX_STAKER_STRATEGY_LIST_LENGTH"
             );
             stakerStrategyList[staker].push(strategy);
         }
 
-        // add the returned shares to their existing shares for this strategy
-        stakerStrategyShares[staker][strategy] += shares;
+        // add the returned scaled shares to their existing scaled shares for this strategy
+        stakerStrategyScaledShares[staker][strategy] += scaledShares;
```

Comment:
can we call back to the DelegationManager here to increase shares?

Reply:
Now I'm wondering if we should just store `stakerStrategyScaledShares` and `podOwnerScaledShares` in the DelegationManager if all deposits are going to require calling the DelegationManager to scale regardless if the staker is delegated or not hmmm